src/main/java/seedu/address/model/property/client/Client.java

```diff
@@ -57,12 +57,36 @@ public boolean isSameClient(Client otherClient) {
      * Converts toString of client back to Client object.
      */
     public static Client fromStringToClient(String toString) {
-        String[] clientInfo = toString.split(";");
-        String name = clientInfo[0].split("Client Name: ")[1];
-        String contact = clientInfo[1].split("Client Contact: ")[1];
-        String email = clientInfo[2].split("Client Email: ")[1];
-        String price = clientInfo[3].split("Client Asking Price: ")[1];
-        return new Client(new Name(name), new Contact(contact), new Email(email), new AskingPrice(price));
+        Name name = null;
+        Contact contact = null;
+        Email email = null;
+        AskingPrice price = null;
```

Comment:
a suggestion bc I don't think its good to initialise the variables to null:
Name name;
Contact contact; etc

Reply:
IDE complains that the variable might not be initialized if i dont put leh

Reply:
oh I see, because the toString may not contain the variable. ok nvrm then haha